26-register_registration/275-creator_navigates_to_files_and_license_tab.feature

```diff
@@ -23,16 +24,19 @@ Feature: Creator navigates to Files and License tab
     When they enter "<Link>" in the Linked Resource field
     And they click the Add Link Button
     Then the Link is listed under Linked Resources
-    And they see that the Title is "<Title>"
-    And they see that the Description is "<Description>"
-    And they see that the Image is "<Image>"
     And they see that the URL is "<URL>"
-    And they see that the Site Name is "<SiteName>"
     And they see a Button to remove the Link
     Examples:
-      | Link                                      | Title                                                  | Description                                                                                                                                                                     | Image                                                                                                                      | URL                                       | SiteName |
-      | https://github.com/BIBSYSDEV/NVA-Frontend | BIBSYSDEV/NVA-Frontend                                 | Web app for NVA. Contribute to BIBSYSDEV/NVA-Frontend development by creating an account on GitHub.                                                                             | https://opengraph.githubassets.com/9b8b6574086bddaac12ee7cbdde3aeea1bed7d3bbf93b5efe34ea14cb438fc5d/BIBSYSDEV/NVA-Frontend | https://github.com/BIBSYSDEV/NVA-Frontend | GitHub   |
-      | https://www.nrk.no/                       | NRK.no – nyheter, tv og radio fra Norge og hele verden | NRK.no er Norges største tilbud på nett: nyheter fra Norge og verden, lokalnyheter, radio- og tv-program, podcast, vær, helse-, kultur-, underholdning-, humor- og debattstoff. | https://gfx.nrk.no/MypkihdsBkCYb-cXvwW8BgLqqp7OO7Fkyj8B_mXoY4Ew                                                            | https://www.nrk.no/                       | NRK      |

```

Comment:
Stemmer det at Title og Description ikke skal vises @garshol ?
Dette var vel lagt inn tidligere for noe forskningsdata (datasett?)

Reply:
ja, endring av desing